### DISCUSSION TITLE
Falling at the first hurdle with Windows 11 and Version 0.10.0

### DISCUSSION THREAD

**Discussion by SirClickALot** [typst/typst#3377] [General]
### Description

I just downloaded the latest release for Windows and UNZipped.
I see the the .exe but when I launch it I see only a 'flash' of command window and nothing else.
Should I be doing something different?

Thank you

### Operating system

Windows

### Typst version

- [X] I am using the latest version of Typst

**Comment by laurmaedje** [comment]
When double-clicking an exe, Windows opens a terminal only for the duration the program runs (which is very short as it just prints a help). To use the Typst CLI, ideally put it in some directory that is added to your PATH environment variable, then open a new terminal manually and type `typst`. It will then print a help of what you can do.

If you just want to start writing, it could be simpler to use the web app or to install Typst LSP and Typst Preview in Visual Studio Code.

**Comment by VibrantTreeStump** [comment]
If you don't want to handle downloading the .exe and adding it to the PATH manually, you can also just install Typst via `winget`, which will handle this for you. To install, open up cmd or PowerShell and execute this command:
`winget install -e --id Typst.Typst`
